enhancements/kube-apiserver/stability-api-removal-notifications.md

```diff
@@ -112,9 +134,66 @@ Value of `removed_release`:
 | 4.9               | 1.23            |                       |
 | 4.10 / 4.10 EUS   | 1.24            | TBD                   |
 
+#### DeprecatedAPIRequest Resource
+
+Patch the apiserver such that requests to a deprecated API are logged to a `DeprecatedAPIRequest` resource corresponding
+to the deprecated API. This resource can help customers determine which workloads are using deprecated APIs, including
+those which are a being removed in the next release.
+
+```go
+package v1alpha1
```

Comment:
given new rules, we cannot make this v1alpha1 and enable it by default.  I think we end up married to this API